indexer/src/main/java/au/org/aodn/esindexer/service/StacCollectionMapperService.java

```diff
@@ -299,6 +287,26 @@ List<Map<String,String>> mapSummariesTemporal(MDMetadataType source) {
         return null;
     }
 
+    @Named("mapSummaries.creation")
+    String mapSummariesCreation(MDMetadataType source) {
+        var dateSources = MapperUtils.findMDDateInfo(source);
+        HashMap<String, String> dateMap = getMetadataDateInfoFrom(dateSources);
+        if (!dateMap.isEmpty()) {
+            return dateMap.get("creation");
+        }
+        return null;
+    }
+
+    @Named("mapSummaries.revision")
+    String mapSummariesRevision(MDMetadataType source) {
+        var dateSources = MapperUtils.findMDDateInfo(source);
+        HashMap<String, String> dateMap = getMetadataDateInfoFrom(dateSources);
+        if (!dateMap.isEmpty()) {
```

Comment:
Same as above the map can be not empty but the get can return null? Better to not check empty but keyExist